Makefile

```diff
@@ -149,6 +149,7 @@ lima-install: manifests
 	helm template \
 		--set controller.enableSafeguards=false \
 		--set controller.expiredDisruptionGCDelay=${EXPIRED_DISRUPTION_GC_DELAY} \
+		--values ./chart/values/dev.yaml \
```

Comment:
WDYT of moving the `controller.enableSafeguards=false` to this dedicated values file?